migrate-profile.md

```diff
@@ -0,0 +1,54 @@
+# Migrate Tanzu Application Platform Profiles
+
+This document describes how to migrate from one Tanzu Application Platform profile to another.
+
+You can perform fresh install of Tanzu Application Platform 1.1 by following the instructions in [Installing Tanzu Application Platform](install-intro.md).
+
+## <a id='prereqs'></a> Prerequisites
+
+Before you migrate from one Tanzu Application Platform to another:
+
+- For information about installing Tanzu Application Platform, see [Install your Tanzu Application Platform profile](install.md#install-profile)
+- For information about installing or updating the Tanzu CLI and plug-ins, see [Install or update the Tanzu CLI and plug-ins](install-tanzu-cli.md#cli-and-plugin)
+- For information on Tanzu Application Platform GUI considerations, see [Tanzu Application Platform GUI Considerations](tap-gui/upgrades.md#considerations)
+- Verify all packages are reconciled by running `tanzu package installed list -A`
```

Comment:
I don't love the guidance on lines 11-14 as they don't make it clear exactly what it is I'm "Required" to do here.

I might expect the list to be something like: (ala trying to be more specific in the guidance)

```
- Ensure you have the most current Tanzu CLI installed.
- Do some sort of comparison between the profile you currently have installed with what comes installed in the new package. Take note.
    - In the case you're adding or removing TAP GUI please see the GUI considerations
```

Reply:
+1